test/uk/gov/hmrc/economiccrimelevyregistration/services/RegistrationValidationServiceSpec.scala

```diff
@@ -500,5 +500,33 @@ class RegistrationValidationServiceSpec extends SpecBase {
         result.isValid shouldBe true
         result         shouldBe Valid(Left(trustRegistration.registration))
     }
+
+    "return the registration if the registration for a non-Uk establishment is valid" in forAll {
+      (validNonUkEstablishmentRegistration: ValidNonUkEstablishmentRegistration) =>
+        val result = service.validateRegistration(validNonUkEstablishmentRegistration.registration)
+        result shouldBe Valid(Right(validNonUkEstablishmentRegistration.registration))
+    }
+
+    "return errors if the registration for a non-UK establishment is invalid" in forAll {
```

Comment:
I would add 2 more test cases here. 

One where utrType is present but UK UTR that matches that type has not been passed. (you can add just one, no need for both)

One where utrType is not passed but either of UK UTRs is passed.